premailer/element.go

```diff
@@ -14,6 +14,7 @@ type elementRules struct {
 	element         *goquery.Selection
 	rules           []*styleRule
 	cssToAttributes bool
+	keepBangImportant bool
```

Comment:
Do we need this flag or should we always keep "important"?

Reply:
I'm not sure actually. I added it as a flag so it wouldn't change functionality for anyone already using the code.

Reply:
Ok, not me either so let's keep it that way